src/gui/mzroll/mzfileio.cpp

```diff
@@ -1322,9 +1342,7 @@ PeakGroup* mzFileIO::readGroupXML(QXmlStreamReader& xml, PeakGroup* parent)
         make_shared<MavenParameters>(*_mainwindow->mavenParameters),
```

Comment:
Any plans to drop mzroll support completely? just curious

Reply:
I had planned on dropping it in this release itself - too many open issues around mzroll import itself. But I am short on time. So maybe later.